contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -608,6 +610,9 @@ public void run() throws Exception {
                     verifyOrAddJsonStore(url);
                 }
                 if (build) {
+                    log.info("==================================================================================");
```

Comment:
These separators are temporary, and eventually will be removed. But that's a good point! I'll keep atomic in mind. Thanks.